electrum/lnworker.py

```diff
@@ -169,18 +170,25 @@ class PaymentInfo(NamedTuple):
     status: int
 
 
-class RecvMPPResolution(Enum):
-    WAITING = enum.auto()
-    EXPIRED = enum.auto()
-    ACCEPTED = enum.auto()
-    FAILED = enum.auto()
```

Comment:
We should add a comment here about these values being persisted in the wallet file, and hence being sensitive to changes. As in:
https://github.com/spesmilo/electrum/blob/02a9ab80be819468352f773bd7120421868d38f0/electrum/lnchannel.py#L77-L79
Btw, like with the channel states, we could potentially persist them by name instead of value. It only matters if we end up changing this enum, and how often. The ChannelState enum has been changed several times and hence in that case it was quite helpful to use the names.